src/CoreWCF.CrankTests/CoreWCFPerfClient/Program.cs

```diff
@@ -0,0 +1,169 @@
+                        catch (Exception ex)
+                        {
+                            Console.WriteLine($"Client Close failed : {ex}.");
+                            ((IClientChannel)client).Abort();
+                        }
+                        requestTime = 0;
+                    }
+
+                    if (((IClientChannel)client).State != CommunicationState.Opened)
+                    {
+                        client = factory.CreateChannel();
+                        await Task.Factory.FromAsync(((IClientChannel)client).BeginOpen, ((IClientChannel)client).EndOpen, null);
+                    }
+                }
+                factory.Close();
+                Console.WriteLine("ChannelFactory Closed.");
+
+                BenchmarksEventSource.Measure("corewcfperf/requests", request);
+                BenchmarksEventSource.Measure("corewcfperf/rps/max", request / test._paramPerfMeasurementDuration.TotalSeconds);
+            }
+        }
+
+        private bool ProcessRunOptions(string[] args)
+        {
+            foreach (string s in args)
+            {
+                Console.WriteLine(s);
+                string[] p = s.Split(new char[] { ':' }, count: 2);
+                if (p.Length != 2)
+                {
+                    continue;
+                }
+
+                switch (p[0].ToLower())
+                {
+                    case Parameters.Binding:
+                        if (!Enum.TryParse<TestBinding>(p[1], ignoreCase: true, result: out _paramBinding))
+                        {
+                            return ReportWrongArgument(s);
+                        }
+                        break;
+
+                    case Parameters.PerfMeasurementDuration:
+                        int perfPerfMeasurementDurationSeconds = 0;
+                        if (!Int32.TryParse(p[1], out perfPerfMeasurementDurationSeconds))
+                        {
+                            return ReportWrongArgument(s);
+                        }
+                        _paramPerfMeasurementDuration = TimeSpan.FromSeconds(perfPerfMeasurementDurationSeconds);
+                        break;
+
+                    case Parameters.Requests:
+                        if (!Int32.TryParse(p[1], out _paramRequests))
+                        {
+                            return ReportWrongArgument(s);
+                        }
+                        break;
+
+                    case Parameters.ServiceUrl:
+                        _paramServiceUrl = p[1];
+                        break;
+
+                    case Parameters.CloseAndOpen:
+                        if (!bool.TryParse(p[1], out _paramCloseAndOpen))
+                        {
+                            return ReportWrongArgument(s);
+                        }
+                        break;
+
+                    default:
+                        Console.WriteLine("unknown argument: " + s);
+                        continue;
```

Comment:
You already have a ReportWrongArguments method. This will again ignore a bad argument and keep going which isn't what's needed.